blog.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Blog</title>
+  </head>
+
+  <body>
+    <header>
+      <img src="images/cart-icon.svg" alt="white cart icon" style="width:10%">
+      <p>$1234.56</p>
+      <p><a href="checkout.html">Checkout</a></p>
+      <img src="images/adoptadog-logo.svg" alt="Adoptadog logo with dog in white" style="width:15%">
+      <h1>Seattle's premiere dog adoption agency</h1>
+      <nav>
+        <a href="index.html">Home</a> /
+        <a href="dogs.html">Dogs</a> /
+        Blog
+      </nav>
+    </header>
+    <h2>Adoptadog blog</h2>
```

Comment:
I'd consider moving into the section element to avoid the html validator warning. 
https://www.w3.org/wiki/HTML/Usage/Headings/Missing